src/components/anchor-navigation/anchor-navigation.test.tsx

```diff
@@ -0,0 +1,364 @@
+import React from "react";
+import { render, screen, act } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import Textbox from "../textbox";
+import {
+  AnchorNavigation,
+  AnchorNavigationItem,
+  AnchorSectionDivider,
+} from ".";
+import { AnchorNavigationProps } from "./anchor-navigation.component";
```

Comment:
**suggestion**: we can import `AnchorNavigationProps` from the index above, reducing the need for an additional import

Reply:
yeah, I missed this as I didn't even look at the imports (other than the bits I had to change of course)